crates/driver/src/boundary/settlement.rs

```diff
@@ -158,10 +159,19 @@ impl Settlement {
             );
         }
 
+        settlement.score = match solution.score().clone() {
+            competition::Score::Solver(score) => http_solver::model::Score::Solver { score },
+            competition::Score::RiskAdjusted(success_probability) => {
+                http_solver::model::Score::RiskAdjusted {
+                    success_probability,
+                    gas_amount: None,
```

Comment:
Can we create a few clean-up items (including getting rid of gas_amount here) to track what we want to remove once solvers are moved over?

Reply:
I captured `gas_amount` removal [here](https://github.com/cowprotocol/services/issues/1897).
I captured `Score::Discount` removal [here](https://github.com/cowprotocol/services/issues/1898).